### ISSUE TITLE
Stub files - Auto-import shown for already imported symbols

### ISSUE THREAD

**Issue by cdce8p** [microsoft/pylance-release#685]
## Environment data

-   Language Server version: 2020.12.0
-   OS and version: macOS
-   Python version: 3.9

## Expected behaviour

The `Auto-import` note should not be displayed for symbols which are already imported.

## Actual behaviour

![Screen Shot 2020-12-04 at 02 05 58](https://user-images.githubusercontent.com/30130371/101109089-59280300-35d6-11eb-8d09-8671f55ba2fe.png)

## Code Snippet / Additional information

From my testing, this seem to affect only stub files.

```python
from typing import Union

Union
```

--

Same example in a normal python file

![Screen Shot 2020-12-04 at 02 18 06](https://user-images.githubusercontent.com/30130371/101109383-13b80580-35d7-11eb-9cfc-4a82da800cab.png)

**Comment by heejaechang** [comment]
@cdce8p can you give me more context/repro on this? creating blank a.py file and testing it doesn't repro it so it feels like it requires some setup to repro?

**Comment by cdce8p** [comment]
@heejaechang I can still reproduce the error with pylance `2021.3.1` in an empty project. However, as stated in the title and description, this only happens for stub files `.pyi`. Normal `.py` files are fine which I find a bit curious.

Here is what I did:
1. Create a new folder
2. Create `a.pyi` with
```py
from typing import Union
```
3. Start typing: `Union` in a new line and activate autocompletion

My folder settings:
```json
{
    "python.languageServer": "Pylance",
    "python.analysis.typeCheckingMode": "strict",
}
```

**Comment by jakebailey** [comment]
I experienced this when I was working on the matplotlib stubs last week; I think the completion provider is using the wrong symbol table when offering completions within stubs, using the one that contains symbols  _externally_ visible to other files, and not the table that is _internally_ visible to the stub itself.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202132-17-march-2021
